chia/rpc/wallet_rpc_api.py

```diff
@@ -500,6 +503,62 @@ async def get_transactions(self, request: Dict) -> Dict:
             "wallet_id": wallet_id,
         }
 
+    def __get_csv_line(self, tx, prefix, date_fmt, delim):
```

Comment:
We need type annotations here. For example:
tx: TransactionRecord, prefix: str, etc

Reply:
will do